index.html

```diff
@@ -28,9 +28,223 @@ <h1 class="homepage-heading">Emmanuel <br>
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="projects-project5"> <img src="images/Rectangle216.png" alt="" class="projects-project5"></div>
+            <div class="projects-img2">
+                <h3 class="heading-blue-background">Simple Shell Project</h3>
+                <div class="buttons">
+                    <ul>
+                        <li><button class="button1">HTML/CSS</button></li>
+                        <li><button class="button2">Ruby on Rails</button></li>
+                        <li><button class="button3">JavaScript</button></li>
+                    </div>
+                <div class=rectangle-and-arrow>
+                    <img class="rectangle" src="images/Rectangle20.png" alt="Rectangle">
+                    <div class="union">
+                    <img class="arrow" src="images/Union.png">
+                    </div>
+                    <div class="see-project">
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="projects-project6"> <img src="images/Rectangle217.png" alt="" class="projects-project6"></div>
+            <div class="projects-img2">
+                <h3 class="heading-blue-background">Variadic functions Projects</h3>
+                <div class="buttons">                    
+                    <ul>
+                        <li><button class="button1">HTML/CSS</button></li>
+                        <li><button class="button2">Ruby on Rails</button></li>
+                        <li><button class="button3">JavaScript</button></li>
+                </div>
+                <div class=rectangle-and-arrow>
+                    <img class="rectangle" src="images/Rectangle20.png" alt="Rectangle">
+                    <div class="union">
+                    <img class="arrow" src="images/Union.png">
+                    </div>
+                    <div class="see-project">
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="About">
+            <div class="about-container">
+                <div class="about-heading">
+                    <h1 class="heading-about">About<h1>
+                </div>
+                <div class="about-me">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+                </div>
+                    <div class= about-collaboration>
+                        <p class="about-start-collab"> Get my Cv <img class="arrow2" src="images/icons8-share-50.png" alt="union"><p>
+                        </div>
```

Comment:
- [ ] Great job implementing changes required by the first reviewer. However, the download arrow always faces downward, while the upload arrow always faces upward. Kindly change the arrow beside your `Get my cv` text to downward arrow in order to match what we have on the Figma design.

### Your Design
![your-arrow](https://user-images.githubusercontent.com/26597432/221376273-9d107d4e-ff5a-4f5a-854f-1e9d8ab6917a.png)

### Figma Design
![temp-arrow](https://user-images.githubusercontent.com/26597432/221376280-e665026a-3a69-45b8-8554-0ce43b9152a0.png)